src/components/TagFilterForm/TagFilterForm.js

```diff
@@ -122,7 +120,7 @@ class TagFilterForm extends Component {
               <TagContainer
                 id={tag}
                 showDelete
-                onDeleteClicked={() => removeTag(tag)}
+                onDeleteClicked={() => {removeTag(tag); this.props.setFlipMove(false);}}
```

Comment:
Could you move this function into a component instance method? (Might need to curry the `tag` parameter.)